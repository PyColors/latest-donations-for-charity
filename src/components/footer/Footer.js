import React, { Component } from "react";
import Appbar from "muicss/lib/react/appbar";

class Footer extends Component {
  render() {
    return (
      <footer className="footer">
        <div className="mui-container mui--text-center">
          Made with â™¥ by <a className="footer__link" href="https://www.pycolors.com">Py Colors</a>
          <br /> <br />
        </div>
      </footer>
    );
  }
}

export default Footer;
