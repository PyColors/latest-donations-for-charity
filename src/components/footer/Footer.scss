.footer {
    color: $primary-color;
    &__link {
        color: $white;
    }

}

