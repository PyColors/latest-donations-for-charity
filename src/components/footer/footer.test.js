import React from "react";
import Footer from "./Footer";

test("Render a div", () => {
  const wrapper = shallow(<Footer>Hello Jest!</Footer>);
  expect(wrapper).toMatchSnapshot();
});
