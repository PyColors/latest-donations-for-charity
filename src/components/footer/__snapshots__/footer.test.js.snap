// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render a div 1`] = `
<footer
  className="footer"
>
  <div
    className="mui-container mui--text-center"
  >
    Made with â™¥ by 
    <a
      className="footer__link"
      href="https://www.pycolors.com"
    >
      Py Colors
    </a>
    <br />
     
    <br />
  </div>
</footer>
`;
